src/components/ArticleId.jsx

```diff
@@ -0,0 +1,75 @@
+import "../App.css"
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import Loading from "./Loading";
+
+function ArticleId() {
+
+    const { article_id } = useParams();
+    const [article, setArticle] = useState({})
+    const [comments, setComments] = useState([])
+    const [isLoading, setIsLoading] = useState(false)
+    const [isCommentsLoading, setIsCommentsLoading] = useState(false)
+    useEffect(() => {
+        setIsLoading(true)
+        fetch(`https://news-backend-vnab.onrender.com/api/articles/${article_id}`)
+            .then(result => {
```

Comment:
You should extract all of your api calls to a utils/api.js module

Reply:
THanks.